components/typeform/actions/update-dropdown-multiple-choice-ranking/update-dropdown-multiple-choice-ranking.mjs

```diff
@@ -59,10 +59,6 @@ export default {
     },
```

Comment:
The description for the **Update Dropdown, Multiple Choice or Ranking** currently says
```
"Update Dropdown, Multiple Choice or Ranking Field Types."
```

I think something like
```
"Update a dropdown, multiple choice, or ranking field.
```
or 
```
"Update a dropdown, multiple choice, or ranking field's choices."
```
would make the action's effect clearer. And in most existing components, it seems like common nouns in `description`s aren't capitalized like in `name`s or titles.